docs/faq/general.txt

```diff
@@ -85,13 +85,6 @@ Django includes code from the Python standard library. Python is distributed
 under a permissive open source license. :source:`A copy of the Python license
 <LICENSE.python>` is included with Django for compliance with Python's terms.
 
-Which sites use Django?
-=======================
-
-`DjangoSites.org`_ features a constantly growing list of Django-powered sites.
-
-.. _DjangoSites.org: https://djangosites.org
```

Comment:
Based off this discussion: https://forum.djangoproject.com/t/replace-django-powered-sites-link-in-community-sections/27562 and the PR to djangoproject.com: https://github.com/django/djangoproject.com/pull/1471/, I think we want to replace DjangoSites.org with https://builtwithdjango.com/projects/
What do you think?